packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -37,13 +38,15 @@ export class Meter implements types.Meter {
   private readonly _metrics = new Map<string, Metric<BaseBoundInstrument>>();
   private readonly _batcher: Batcher;
   readonly labels = Meter.labels;
+  readonly resource: Resource;
```

Comment:
```suggestion
  private readonly _resource: Resource;
```

Reply:
I would have accepted this suggestion, but it required a few additional changes.